### ISSUE TITLE
Not upgrading str.format() when statement on multiple lines

### ISSUE THREAD

**Issue by heidonomm** [ikamensh/flynt#112]
Flynt does not convert strings when the statement is on multiple lines.
example:

```python
text = 'text'
placeholder = 'placeholder'

'some {} here as {}'.format(
text, placeholder)
```
should be converted to f'some {text} here as {placeholder}' but is not.

**Comment by ikamensh** [comment]
Hi, I could reproduce the issue. Will update once it's fixed.

**Comment by ikamensh** [comment]
This no longer occurs, maybe due to fixes on #114 or #115. 
Added a unit test to prevent regressions.

Let me know if it still occurs as of flynt v0.74

**Comment by ikamensh** [comment]
I can't reproduce the problem. 

```
~/tmp $ flynt --version                                                        (base) 10:31:56
0.76
~/tmp $ python -V                                                              (base) 10:32:38
Python 3.8.8
~/tmp $ echo "'my string {}'.format(                                           (base) 10:32:42
            my_var,
        )" > tst.py
~/tmp $ cat tst.py                                                             (base) 10:33:04
'my string {}'.format(
    my_var,
)
~/tmp $ flynt tst.py                                                           (base) 10:33:06
Running flynt v.0.76

Flynt run has finished. Stats:

Execution time:                            0.004s
Files checked:                             1
Files modified:                            1
Character count reduction:                 15 (40.54%)

Per expression type:
No old style (`%`) expressions attempted.
`.format(...)` calls attempted:            1/1 (100.0%)
No concatenations attempted.
F-string expressions created:              1

_-_._-_._-_._-_._-_._-_._-_._-_._-_._-_._-_._-_._-_._-_._-_._-_._-_._-_._-_._-_._-_._-_._-_._-_._-_.
Please run your tests before committing. Did flynt get a perfect conversion? give it a star at:
~ https://github.com/ikamensh/flynt ~
Thank you for using flynt. Upgrade more projects and recommend it to your colleagues!

~/tmp $ cat tst.py                                                             (base) 10:33:12
f'my string {my_var}'
~/tmp $
```
